src/storage/GetUUIDProcessor.h

```diff
@@ -0,0 +1,76 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#ifndef STORAGE_GETUUIDPROCESSOR_H_
+#define STORAGE_GETUUIDPROCESSOR_H_
+
+#include "base/Base.h"
+#include "base/MurmurHash2.h"
+#include "storage/BaseProcessor.h"
+#include "kvstore/NebulaStore.h"
+#include "time/WallClock.h"
+
+namespace nebula {
+namespace storage {
+
+class GetUUIDProcessor : public BaseProcessor<cpp2::GetUUIDResp> {
+public:
+    static GetUUIDProcessor* instance(kvstore::KVStore* kvstore) {
+        return new GetUUIDProcessor(kvstore);
+    }
+
+    void process(const cpp2::GetUUIDReq& req) {
+        constexpr size_t hashMask = 0xFFFFFFFF00000000;
+        constexpr size_t timeMask = 0x00000000FFFFFFFF;
+        CHECK_NOTNULL(kvstore_);
+        auto spaceId = req.get_space_id();
+        auto partId = req.get_part_id();
+        auto name = req.get_name();
+        auto key = NebulaKeyUtils::uuidKey(partId, name.c_str());
+        std::string val;
+        VertexID vId;
+        auto ret = kvstore_->get(spaceId, partId, key, &val);
+        // try to get the corresponding vertex id
+        if (ret != kvstore::ResultCode::SUCCEEDED) {
+            // need to generate new vertex id of this uuid
+            MurmurHash2 hashFunc;
+            auto hashValue = hashFunc(name);
+            auto now = time::WallClock::fastNowInMicroSec();
```

Comment:
for int32 we’d better use seconds

Reply:
If we use second, I suppose we would have high conflict rate? Cuz If we generate many uuid in a partition at the exact same second, the time will be all the same.

Reply:
we can’t use int32 to represent micro seconds, it has more conflicts

Reply:
Ok